src/Parsers/examples/YAML_fuzzer.cpp

```diff
@@ -0,0 +1,38 @@
+#include <iostream>
+#include <fstream>
+#include <string>
+#include <cstdio>
+#include <time.h>
+#include <filesystem>
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t * data, size_t size)
+{
+    /// How to test:
+    /// build ClickHouse with YAML_fuzzer.cpp
+    /// ./YAML_fuzzer YAML_CORPUS
+    /// where YAML_CORPUS is a directory with different YAML configs for libfuzzer
+    char* file_name = std::tmpnam(nullptr);
```

Comment:
`std::tmpnam(nullptr)` returns a pointer to an internal static buffer, it's rather a thread-unsafe way, better not to use this form.